Makefile

```diff
@@ -0,0 +1,14 @@
+# Why is there a `Makefile` when we already have a `Fastfile` in this project?
+#
+# Because when running Ruby code (i.e. `bundle exec fastlane <lane>`) from
+# within an Xcode build phase, Xcode will use a shell different from the user's
+# and with system Ruby where Bundler and Fastlane may or may not be available.
+# `make` on the other hand, is always available, therefore `make lint` can be
+# reliably called from both the user's terminal, Xcode, and CI.
+SWIFTLINT=./Pods/SwiftLint/swiftlint
+
+lint:
+	$(SWIFTLINT) lint --quiet
+
+format:
+	$(SWIFTLINT) lint --autocorrect --quiet
```

Comment:
The approach presented here might be a bit controversial, but is working well for the Pocket Casts iOS devs.

I don't like the overhead of having `Makefile` and `Fastfile` but has one positive aspect. Our long term vision is to remove the need to deal with Ruby for feature devs, but they might still need to run CLI tasks such as these and `make` is an excellent solution for that. Introducing `make` now could pave the way for leaning more into it in the future.